app/services/user/service.go

```diff
@@ -94,6 +96,19 @@ func (u *userService) SignUpUser(ctx context.Context, signUpParams *dto.UserSign
 	}, nil
 }
 
+func (u *userService) UpdateUser(ctx context.Context, updateParams *userRepo.User) error {
+	return u.userRepo.UpdateUser(ctx, &userRepo.UpdateUserParams{
+		ID:           updateParams.ID,
+		FirstName:    updateParams.FirstName,
+		LastName:     updateParams.LastName,
+		Email:        updateParams.Email,
+		PhoneNumber:  updateParams.PhoneNumber,
+		PasswordHash: updateParams.PasswordHash,
```

Comment:
Same as the comment above about needing to double check users permissions before changing password